back/spec/models/group_spec.rb

```diff
@@ -69,4 +69,13 @@
       group.update_memberships_count!
     end
   end
+
+  describe 'generate_slug!' do
```

Comment:
Name of the method doesn't have a bang
```suggestion
  describe 'generate_slug' do
```

Reply:
Good catch